x11-wm/i3-gaps/i3-gaps-4.19.ebuild

```diff
@@ -0,0 +1,93 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson
+
+DESCRIPTION="i3 fork with gaps and some more features"
+HOMEPAGE="https://github.com/Airblader/i3"
+SRC_URI="https://github.com/Airblader/i3/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+IUSE="doc test"
+RESTRICT="!test? ( test )"
+
+CDEPEND="dev-libs/glib:2
+	dev-libs/libev
+	dev-libs/libpcre
+	dev-libs/yajl
+	x11-libs/cairo[X,xcb(+)]
+	x11-libs/libxcb[xkb]
+	x11-libs/libxkbcommon[X]
+	x11-libs/pango[X]
+	x11-libs/startup-notification
+	x11-libs/xcb-util
+	x11-libs/xcb-util-cursor
+	x11-libs/xcb-util-keysyms
+	x11-libs/xcb-util-wm
+	x11-libs/xcb-util-xrm
+"
+DEPEND="${CDEPEND}
+	test? (
+		dev-perl/ExtUtils-PkgConfig
+		dev-perl/IPC-Run
+		dev-perl/Inline
+		dev-perl/Inline-C
+		dev-perl/X11-XCB
+		dev-perl/XS-Object-Magic
+		x11-apps/xhost
+		x11-base/xorg-server[xephyr,xvfb]
+		x11-misc/xvfb-run
+	)
+"
+BDEPEND="app-text/asciidoc
+	app-text/xmlto
+	dev-lang/perl
+	virtual/pkgconfig
+"
+RDEPEND="${CDEPEND}
+	dev-lang/perl
+	dev-perl/AnyEvent-I3
+	dev-perl/JSON-XS
+	!x11-wm/i3
+"
+
+S="${WORKDIR}/i3-${PV}"
+
+DOCS=( RELEASE-NOTES-$(ver_cut 1-3) )
+
+PATCHES=(
+	"${FILESDIR}/${PN}-4.18-musl.patch"
+)
+
+src_configure() {
+	local emesonargs=(
+		-Ddocdir="/usr/share/doc/${P}"
```

Comment:
```
 * Messages for package x11-wm/i3-gaps-4.19:

 * The ebuild is installing to one or more unexpected paths:
 * 
 *   /usr/share/doc/i3
 * 
 * Please fix the ebuild to use correct FHS/Gentoo policy paths.
```

We use `${PF}` for doc dir, however it doesn't seem to be problem here. This installs an empty dir called `/usr/share/doc/i3`.

Reply:
Reported upstream and I've just pushed a patch here for this ebuild.

https://github.com/i3/i3/issues/4268